### DISCUSSION TITLE
With PHP 8 attributes, the "Attribute" rule seems a bit off

### DISCUSSION THREAD

**Discussion by henriquemoody** [Respect/Validation#1415] [General]
I wonder if the rule becomes a bit weird because of its name and its exceptions message.

What are your thoughts?

**Comment by alganet** [comment]
Yep, we should rename it to `property` or something like that in a future release.

**Reply by henriquemoody** [reply]
Are you planning a MAJOR or MINOR next release?

**Reply by alganet** [reply]
I don't know yet. I have some changes that break BC in mind, so it will be probably a major.

**Reply by henriquemoody** [reply]
I was asking to know whether it would be better to create a `Property` one and deprecate `Attribute` or simply rename it. I would prefer the first option.
